src/transformStateWithClones.js

```diff
@@ -8,6 +8,39 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const stateCopy = [];
+  const transformedState = { ...state };
+  const addProperties = 'addProperties';
+  const removeProperties = 'removeProperties';
+  const clear = 'clear';
```

Comment:
this variables is redundant here. You can use `enum` instead

Reply:
Ok. Thanks.